### ISSUE TITLE
Bulk Data export does not include system/Patient.read scope (FI-1392)

### ISSUE THREAD

**Issue by medhost-bfindeisen** [onc-healthit/inferno-program#361]
Thank you for reporting a possible bug in Inferno! Please fill in as much of the template below as you can.

**Subject of the issue**
The default scope list of bulk data export does not include the system/Patient.read scope

https://chat.fhir.org/#narrow/stream/179309-inferno/topic/Bulk.20Data.20Scope.20List

**Your environment**
* Edition of inferno (Community or Program): Program
* Version of inferno: 1.6.2
* Which browser and version(s) is the bug present on?: Chrome

**Steps to reproduce**

1. Start testing
2. Select Multi-Patient API
3. Notice Bulk Data Scopes does not include system/Patient.read

**Expected behavior**
system/Patient.read scope be included in the list of bulk data scopes

**Actual behavior**
system/Patient.read is not in the list of scopes:
![image](https://user-images.githubusercontent.com/69013924/134024001-d49873a2-ec0a-40cf-973c-e0e0f78216b6.png)

**Comment by arscan** [comment]
Thanks for the report @medhost-bfindeisen.  I confirmed that we do not include that scope by default -- in reviewing the history it appears that it has been incorrect ever since we moved from `system/*.read` to the explicit resource-enumerated scopes by default in this field over a year ago.  Systems can write in their old values here, which is probably why we have missed it for so long, but obviously we want the default to be correct.  We can correct this in the next version.
